content/en/blog/Volcano-1.11.0-release.md

```diff
@@ -0,0 +1,473 @@
+
+For detailed design and user guide, please refer to: **[Cloud Native Colocation | Volcano](https://volcano.sh/en/docs/colocation/)**.
+
+Sincerely thanks to community developer: **@william-wang** for this contribution!
+
+### **Load-Aware Descheduling: Intelligent Cluster Resource Balancing**
+
+In Kubernetes clusters, dynamic workload changes often lead to uneven node resource utilization, causing hotspots that impact cluster stability and efficiency. To address this, Volcano v1.11 introduces **Load-Aware Descheduling**, dynamically adjusting Pod distribution based on real node load to ensure balanced resource utilization and avoid hotspots, improving overall cluster performance and reliability. Load-aware descheduling is incubated in the sub-project: https://github.com/volcano-sh/descheduler.
+
+#### **Core Capabilities:**
+
+- **Load-Aware Scheduling:** Monitors real CPU and memory load metrics to dynamically adjust Pod distribution, avoiding reliance on Pod Request-based scheduling.
+- **Timed and Dynamic Triggers:** Supports CronTab-based or fixed-interval descheduling to adapt to different scenarios.
+
+#### **Use Cases:**
+
+- **Uneven Node Resource Utilization:** Balances node load when some nodes are overutilized while others are underutilized.
+- **Hotspot Node Management:** Migrates Pods from overloaded nodes to ensure stability.
+
+<div style="text-align: center;"> {{<figure library="1" src="./descheduler/descheduler_EN.svg">}}
+</div>
+
+#### **Technical Highlights:**
+
+- **Descheduling Based on Actual Load:**
+
+  Unlike traditional scheduling strategies based on Pod Requests, Volcano's load-aware descheduling is more precise, accurately reflecting the actual resource usage of nodes.
+
+- **Seamless Integration with Kubernetes Ecosystem:**
+
+  Compatible with the native Kubernetes scheduler, enabling load-aware descheduling without requiring additional configurations.
+
+- **Flexible Policy Configuration:**
+
+  Users can customize descheduling intervals or trigger conditions based on business requirements , ensuring flexibility and controllability in scheduling.
+
+For detailed user guide, please refer to: **[Load-aware Descheduling | Volcano](https://volcano.sh/en/docs/descheduler/)**.
+
+Sincerely thanks to community developer: **@Monokaix** for this contribution!
+
+### **Fine-Grained Job Fault Recovery: Efficient Task Interruption Handling**
+
+In AI, Big Data, and HPC scenarios, job stability and fault recovery are critical. Traditional fault recovery strategies often restart entire Jobs when a single Pod fails, wasting resources and potentially restarting training from scratch. With the rise of **checkpointing** and **resume-from-checkpoint** techniques, single Pod failures no longer require full Job restarts. Volcano v1.11 introduces **Fine-Grained Job Fault Recovery ** feature, offering flexible fault handling mechanisms to efficiently manage task interruptions and improve training efficiency.
```

Comment:
![image](https://github.com/user-attachments/assets/48a235fe-3dd3-42e4-9e10-4cc6a21a192f)

`Fine-Grained Job Fault Recovery` didn't not be bolded

Reply:
done